opm/parser/eclipse/RawDeck/RawRecord.hpp

```diff
@@ -34,32 +37,27 @@ namespace Opm {
     public:
         RawRecord(const std::string& singleRecordString, const std::string& fileName = "", const std::string& keywordName = "");
 
-        std::string pop_front();
+        string_view pop_front();
         void push_front(std::string token);
         size_t size() const;
 
         const std::string& getRecordString() const;
-        const std::string& getItem(size_t index) const;
+        string_view getItem(size_t index) const;
         const std::string& getFileName() const;
```

Comment:
If moving the too-many-items handling from `ParserRecord::parse()` to `ParserKeyword::parse()` we can get rid of the `filename` and `keyword` members of the `RawRecord` class. Maybe not much performance to gain - but slightly cleaner.